test/finance.spec.ts

```diff
@@ -480,11 +483,21 @@ describe('finance', () => {
       });
 
       describe('bic()', () => {
-        it('should return a random yet formally correct BIC number', () => {
+        it('should return a BIC number', () => {
           const bic = faker.finance.bic();
 
           expect(bic).toBeTypeOf('string');
-          expect(bic).toMatch(/^[A-Z]{6}[A-Z0-9]{2}([A-Z0-9]{3})?$/);
+          bic.length === 8
+            ? expect(bic).toMatch(/^[A-Z]{6}[A-Z0-9]{2}$/)
+            : expect(bic).toMatch(/^[A-Z]{6}[A-Z0-9]{2}[A-Z0-9]{3}$/);
```

Comment:
```suggestion
          bic.length === 8
            ? expect(bic).toMatch(/^[A-Z]{6}[A-Z0-9]{2}$/)
            : expect(bic).toMatch(/^[A-Z]{6}[A-Z0-9]{2}[A-Z0-9]{3}$/);
```
This is a bit janky, any better way to write this?

Reply:
Wrap the last part in parenthesis and add a question mark behind this to mark it as optional.